src/arc.rs

```diff
@@ -735,3 +735,8 @@ impl<T> AsRef<T> for Arc<T> {
 }
 
 impl<T> Unpin for Arc<T> {}
+
+#[cfg(feature = "enum-ptr")]
+unsafe impl<T> enum_ptr::Aligned for Arc<T> {
```

Comment:
for convenience, this is the link to the relevant safety doc: https://docs.rs/enum-ptr/latest/enum_ptr/trait.Aligned.html#safety